features/me_can_signup_to_application.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Signing up to the account
+  as a user 
+  In order to use the application
+  I would like to login
```

Comment:
Your user story and PR specifies that this is a signup feature, yet you have submitted a feature that tests the login functionality. Please revisit this and make sure it matches the description